ttl/hbp_licence_ontology.ttl

```diff
@@ -0,0 +1,24 @@
+@prefix cc: <http://creativecommons.org/ns#> .
+@prefix nsu: <http://www.FIXME.org/nsupper#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix by-sa: <http://creativecommons.org/licenses/by-sa/> .
+@prefix by-nc-sa: <http://creativecommons.org/licenses/by-nc-sa/> .
```

Comment:
Use capital letters for the prefix names you add to be consistent with others hbp ontologies.